pom.xml

```diff
@@ -475,6 +476,27 @@
         <artifactId>grpc-all</artifactId>
         <version>${grpc.version}</version>
       </dependency>
+      <dependency>
+        <groupId>io.grpc</groupId>
+        <artifactId>grpc-netty-shaded</artifactId>
+        <version>${grpc.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>io.grpc</groupId>
+        <artifactId>grpc-protobuf</artifactId>
+        <version>${grpc.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>io.grpc</groupId>
+        <artifactId>grpc-stub</artifactId>
+        <version>${grpc.version}</version>
+      </dependency>
+      <dependency> <!-- necessary for Java 9+ -->
+        <groupId>org.apache.tomcat</groupId>
+        <artifactId>annotations-api</artifactId>
+        <version>${tomcat-annotations.version}</version>
+        <scope>provided</scope>
+      </dependency>
```

Comment:
There's a grpc-bom Maven BOM available since version 1.25 . I wonder if we should replace the individual dependencies in the `dependencyManagement` of `pom.xml` with the `grpc-bom` import?

Something like:
```
<dependency>
  <groupId>io.grpc</groupId>
  <artifactId>grpc-bom</artifactId>
  <version>${grpc.version}</version>
  <type>pom</type>
  <scope>import</scope>
</dependency>
```

Reply:
This comment looks addressed